saltpack_test.go

```diff
@@ -85,3 +90,31 @@ func TestSaltpackVerifyNoSignature(t *testing.T) {
 	require.Error(t, err, "Should have failed verify")
 	t.Logf("Error: %s", err)
```

Comment:
Maybe instead of logging the error, you can use [`require.EqualError`](https://godoc.org/github.com/jbert/testify/require#EqualError)